wizard.sh

```diff
@@ -0,0 +1,299 @@
+#!/bin/bash
+
+set -e
+
+untracked="(untracked)"
+REPO=$(cd $(dirname $0); pwd)
+LINT="false"
+BUILD="false"
+PUSH_LATEST="false"
+RELEASE=""
+
+debugInfo () {
+  echo "Repo:                     $REPO"
+  echo "Will lint:                $LINT"
+  echo "Will build:               $BUILD"
+  echo "Will release:             $RELEASE"
+  echo "Will push latest docker:  $PUSH_LATEST"
+  echo "Use Docker:               $USE_DOCKER"
+  echo "Is CI:                    $CI"
+}
+
+dockerLogin () {
```

Comment:
This is mostly meant for Travis CI. Locally, the user should handle the login before running the script. This is because a specific version of `docker-credential-pass` is distributed inside `filebrowser/dev`. I.e., on Windows, `dockerLogin` will probably fail. It will also fail in any non-amd64 machine.

So, it is ok to keep this function here. But I think we should add a comment about it.

Reply:
We can also test if is a ci using the `CI` env var. Otherwise we just do nothing on that function. WHat do you think?

Reply:
Agree.

Reply:
CHanged that, please take a look @1138-4EB

Reply:
@1138-4EB maybe if it's not CI, we shouldn't do `docker login` and `docker logout` and just bypass those steps. Probably the user is logged in already. Agree?